### ISSUE TITLE
Users shouldn't be able to use /timeout in race

### ISSUE THREAD

**Issue by bencie** [ddnet/ddnet#8110]
This is probably very niche, but this caused me to lose a run of multiple hours. If you set a /timeout code before you timeout, and you rejoin the server before your timeout protection starts, you will not automatically use your timeout protection, as expected. If you then accidentally type `/timeout code` twice, then you would first use your protection on your main tee, then secondly on your dummy, at which point your main tee disappears as it takes the position of your dummy.

https://github.com/ddnet/ddnet/assets/78111412/b1cfb248-0a70-4866-814f-78da6a18d023

I'd suggest not letting people use /timeout if they already started to race. If you really want to use timeout protection with your main to take your dummy's, you can just kill before using /timeout instead.

**Comment by heinrich5991** [comment]
Please use the built-in timeout randomization feature. It fixes this.

**Comment by bencie** [comment]
Yeah, I agree. I personally have always used /timeout after I encountered a bug a long time ago where I couldn't take my timeout, as I feel safe using it knowing I will always be able to use my protection with the code I specifically set. However, I re-tested it now and couldn't reproduce that bug, so I removed my custom timeout code.
